app/controllers/invitations_controller.rb

```diff
@@ -39,7 +39,8 @@ def create
       InvitationService.invite_to_group(recipient_emails: @invite_people_form.emails_to_invite,
                                         message: @invite_people_form.message_body,
                                         group: @invitable,
-                                        inviter: current_user)
+                                        inviter: current_user,
+                                        subject: params[:invite_people_form][:subject])
```

Comment:
This is unfortunately an email injection risk, I'm pretty sure. 
I'd much prefer if we put the subject into the @invite_people_form and sanitized it there.